### PR TITLE
Uncomment out cold state tests

### PR DESCRIPTION
**Pull request by terencechain**
#5230 was accidentally reviewed and merged with commented cold state tests

This PR has fixed and uncommented out those cold state related tests

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Instead of commenting out this code, please use `t.Skip("reason")` or just delete it. Commented code will never be maintained and will immediately rot.

**Comment by terencechain** [comment]
> Instead of commenting out this code, please use `t.Skip("reason")` or just delete it. Commented code will never be maintained and will immediately rot.

It wasn't meant to be skipped nor commented, the PR just got merged prematurely and I just noticed it today

**Comment by prestonvanloon** [comment]
Oh sorry, I misunderstood the diff and thought you were adding commented code.

File: beacon-chain/state/stategen/cold_test.go [hunk 1]
```diff
@@ -27,281 +30,289 @@ func TestSaveColdState_NonArchivedPoint(t *testing.T) {
...
+	db := testDB.SetupDB(t)
+	defer testDB.TeardownDB(t, db)
+
+	service := New(db, cache.NewStateSummaryCache())
+	if _, err := service.loadColdStateByRoot(ctx, [32]byte{'a'}); err != errUnknownStateSummary {
+		t.Fatal("Did not get correct error")
+	}
+}
+
+func TestLoadColdStateByRoot_ByArchivedPoint(t *testing.T) {
```
Comment (prestonvanloon) [inline beacon-chain/state/stategen/cold_test.go]: This is failing

Comment (terencechain) [reply beacon-chain/state/stategen/cold_test.go]: Thanks. Looking

**Pull request merged**